rs/execution_environment/src/execution_environment/tests.rs

```diff
@@ -886,10 +886,13 @@ fn get_canister_status_of_nonexisting_canister() {
     assert_eq!(ErrorCode::CanisterNotFound, err.code());
 }
 
-fn get_canister_status(test: &mut ExecutionTest, canister_id: CanisterId) -> CanisterStatusResultV2 {
+fn get_canister_status(
+    test: &mut ExecutionTest,
+    canister_id: CanisterId,
+) -> CanisterStatusResultV2 {
     match test.canister_status(canister_id).unwrap() {
         WasmResult::Reply(reply) => CanisterStatusResultV2::decode(&reply).unwrap(),
-        WasmResult::Reject(msg) => panic!("{}", msg) ,
+        WasmResult::Reject(msg) => panic!("{}", msg),
```

Comment:
```suggestion
        WasmResult::Reject(msg) => panic!("unexpected reject: {}", msg),
```
Small nit I noticed.